packages/core/src/createCheckConsistency.ts

```diff
@@ -22,81 +22,68 @@ const defaults = {
 
 /**
  * @method createCheckConsistency
+ * 
+ * @summary Creates a new `checkConsistency()` method, using a custom Provider instance.
  *
  * @memberof module:core
+ * 
+ * @ignore
  *
- * @param {Provider} provider - Network provider
+ * @param {Provider} provider - The Provider object that the method should use to call the node's API endpoints.
  *
- * @return {function} {@link #module_core.checkConsistency `checkConsistency`}
+ * @return {Function} [`checkConsistency`]{@link #module_core.checkConsistency}  - A new `checkConsistency()` function that uses your chosen Provider instance.
  */
 export const createCheckConsistency = ({ send }: Provider) =>
     /**
-     * Checks if a transaction is _consistent_ or a set of transactions are _co-consistent_, by calling
-     * [`checkConsistency`](https://docs.iota.org/iri/api#endpoints/checkConsistency) command.
-     * _Co-consistent_ transactions and the transactions that they approve (directly or inderectly),
-     * are not conflicting with each other and rest of the ledger.
+     * This method finds out if a transaction has a chance of being confirmed, using the connected IRI node's
+     * [`checkConsistency`](https://docs.iota.org/docs/node-software/0.1/iri/references/api-reference#checkconsistency) endpoint.
+     * 
+     * A consistent transaction is one where:
+     * - The node has the transaction's branch and trunk transactions in its ledger
+     * - The transaction's bundle is valid
+     * - The transaction's branch and trunk transactions are valid
+     * 
+     * For more information about what makes a bundles and transactions valid, see [this article](https://docs.iota.org/docs/node-software/0.1/iri/concepts/transaction-validation).
      *
-     * As long as a transaction is consistent it might be accepted by the network.
-     * In case a transaction is inconsistent, it will not be accepted, and a reattachment
-     * is required by calling [`replayBundle`]{@link #module_core.replayBundle}.
+     * As long as a transaction is consistent it has a chance of being confirmed.
+     * 
+     * ## Related methods
+     * 
+     * If a consistent transaction is taking a long time to be confirmed, you can improve its chances, using the [`isPromotable()`]{@link #module_core.isPromotable}
```

Comment:
I don't think [isPromotable](https://github.com/iotaledger/iota.js/blob/next/packages/core/src/createIsPromotable.ts#L26) improves the chances. It is a wrapper around `checkConsistency` which checks if transactions are consistent.